### ISSUE TITLE
local-docker connection not populating dashboard

### ISSUE THREAD

**Issue by jameski83** [WillFantom/sui#4]
I am trying to use your fork with the local-docker provider but it only shows SUI on the homepage. I don't use traefik, will this still work with just local-docker?

**Comment by WillFantom** [comment]
This should work fine with just the `local-docker` provider setup 👍 Although, you should use the `--sui.<>=<>` labels for your containers to enable them and to have SUI show friendly names + use appropriate icons on the dashboard.

**Comment by Code-Slave** [comment]
having the same issue. I have 30 containers on a synolgy nas. Is there specific naming etc i need to do in the containers. it sees docker (confirmed via debug log) but shows nothing

**Comment by WillFantom** [comment]
I'm currently working on a separate version of this that should make configuration far easier, but due to my work this is taking longer than I thought. For now though, I think you might just need to use container labels (in the readme [here](https://github.com/WillFantom/sui#using-docker-labels)). Just having the `sui.enabled=true` label for each container should at least make them show in the dashboard.

**Comment by jameski83** [comment]
Thanks yes it works if I manually add the labels `sui.enabled=true` and `sui.url=xxxx` to each container

**Comment by Code-Slave** [comment]
Same. Although having to add sui.url defaults the whole purpose of autodiscovery i think

**Comment by WillFantom** [comment]
You're right, but the decision was privacy focused (don't have to remember to disable/hide apps). You can modify the json config for the local docker provider adding `"default": true` if you want apps to be enabled by default (I think, I can't test this myself right now). I'll make sure this is a tested feature in the updated version.

**Comment by Seishirou101** [comment]
Adding `"default": true` does not enable them by default sadly.
